events/class_tests/event_tests.py

```diff
@@ -0,0 +1,67 @@
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+from ..models import Event, time_format
+from datetime import datetime
+
+import pytest
+
+DATE_TIME_START = datetime(2022, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
+DATE_TIME_END = datetime(2022, 3, 24, 14, 12, 12, 0, tzinfo=timezone.utc)
+TITLE = 'new_title'
+LOCATION = 'new_location'
+DESCRIPTION = 'new_description'
+
+
+@pytest.fixture
+def new_event():
+    return Event(
+        title=TITLE, location=LOCATION, description=DESCRIPTION,
+        date_time_start=DATE_TIME_START, date_time_end=DATE_TIME_END
+    )
+
+
+@pytest.fixture
+def persist_event(db, new_event):
+    new_event.save()
+    return new_event
+
+
+@pytest.mark.django_db
+def test_add_event(persist_event):
+    assert persist_event in Event.objects.all()
+
+
+@pytest.mark.django_db
+def test_delete_event(persist_event):
+    persist_event.delete()
+    assert persist_event not in Event.objects.all()
+
```

Comment:
Because of the wide use in @pytest.mark.django_db
I would suggest creating a class for all the tests in this file and placing this decorator above the class.
```suggestion
@pytest.mark.django_db
class EventTests():
   pass 
```